src/base/rss/rss_autodownloadrule.cpp

```diff
@@ -117,13 +121,48 @@ namespace RSS
                     && (lastMatch == other.lastMatch)
                     && (savePath == other.savePath)
                     && (category == other.category)
-                    && (addPaused == other.addPaused);
+                    && (addPaused == other.addPaused)
+                    && (smartFilter == other.smartFilter);
         }
     };
 }
 
 using namespace RSS;
 
+QString computeEpisodeName(const QString &article)
+{
+    static const QRegularExpression episodeRegex("(?:^|[^A-Za-z0-9])(?:"
```

Comment:
Please use another indentation. E.g.:
```c++
static const QRegularExpression episodeRegex(
                "(?:^|[^A-Za-z0-9"

                //Format 1: s01e01
                "(?:s(\\d{1,})e(\\d{1,}))|"

                // and so on...
```
You have too much empty space on the left.